src/openrct2/CmdlineSprite.cpp

```diff
@@ -632,25 +633,18 @@ int32_t cmdline_for_sprite(const char** argv, int32_t argc)
                 return -1;
             }
 
-            auto spriteFile = SpriteFile::Open(spriteFilePath);
-            if (!spriteFile.has_value())
-            {
-                fprintf(stderr, "Unable to open sprite file: %s\nCanceling\n", spriteFilePath);
-                return -1;
-            }
-
-            spriteFile->AddImage(importResult.value());
-
-            if (!spriteFile->Save(spriteFilePath))
-            {
-                fprintf(stderr, "Could not save sprite file: %s\nCanceling\n", imagePath.c_str());
-                return -1;
-            }
+            spriteFile.AddImage(importResult.value());
 
             if (!silent)
                 fprintf(stdout, "Added: %s\n", imagePath.c_str());
         }
 
+        if (!spriteFile.Save(spriteFilePath))
+        {
+            fprintf(stderr, "Could not save sprite file: Canceling\n");
```

Comment:
```suggestion
            log_error("Could not save sprite file, cancelling.");
```

Reply:
I'd keep it using fprintf, log_error spits out the line and column and filename.